JASP-Engine/JASP/R/ldBeta.R

```diff
@@ -19,17 +19,17 @@ LDbeta <- function(jaspResults, dataset, options, state=NULL){
   options <- .ldRecodeOptionsBeta(options)
   
   #### Show beta section ----
-  .ldIntroText(jaspResults, options, "beta distribution")
+  .ldIntroText(jaspResults, options, gettext("beta distribution"))
   .ldBetaParsSupportMoments(jaspResults, options)
   
   
-  pdfContainer <- .ldGetPlotContainer(jaspResults, options, "plotPDF", "Probability Density Function", 3)
+  pdfContainer <- .ldGetPlotContainer(jaspResults, options, "plotPDF", gettext("Probability Density Function"), 3)
   .ldFillPDFContainer(pdfContainer, options, .ldFormulaBetaPDF)
   
-  cdfContainer <- .ldGetPlotContainer(jaspResults, options, "plotCDF", "Cumulative Beta Function", 4)
+  cdfContainer <- .ldGetPlotContainer(jaspResults, options, "plotCDF", gettext("Cumulative Beta Function"), 4)
```

Comment:
Cumulative Distribution Function

Reply:
ok AND
-ld Chisq.R
-ldExponential.R
ldGamma.R
ldGammaInverse.R